source/features/show-whitespace.css

```diff
@@ -1,3 +1,32 @@
+/*
+Can't use this because attributes are strings and calc() won't cast them
+ttps://stackoverflow.com/a/40815884
```

Comment:
Should this file get a GHE comment?

Reply:
GHE is mentioned in a different comment. This code is not related to GHE

Reply:
Isn't this link missing a leading `h`?